docs/network protocol.md

```diff
@@ -702,14 +702,17 @@ GameData is a **dict** but contains these keys and values. It's broken out into
 | checksum            | str            | A checksum hash of this game's data.                                                                                          |
 
 ### Tags
-Tags are represented as a list of strings, the common Client tags follow:
-
-| Name       | Notes                                                                                                                                                                                                              |
-|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| AP         | Signifies that this client is a reference client, its usefulness is mostly in debugging to compare client behaviours more easily.                                                                                  |
-| DeathLink  | Client participates in the DeathLink mechanic, therefore will send and receive DeathLink bounce packets                                                                                                            |
-| Tracker    | Tells the server that this client will not send locations and is actually a Tracker. When specified and used with empty or null `game` in [Connect](#connect), game and game's version validation will be skipped. |
-| TextOnly   | Tells the server that this client will not send locations and is intended for chat. When specified and used with empty or null `game` in [Connect](#connect), game and game's version validation will be skipped.  |
+Tags are represented as a list of strings, the common client tags follow:
+
+| Name       | Notes                                                                                                                             |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| AP         | Signifies that this client is a reference client, its usefulness is mostly in debugging to compare client behaviours more easily. |
+| DeathLink  | Client participates in the DeathLink mechanic, therefore will send and receive DeathLink bounce packets.                          |
+| HintGame\* | Indicates the client is a hint game, made to send hints instead of locations. Special join/leave message.                         |
+| Tracker\*  | Indicates the client is a tracker, made to track instead of sending locations. Special join/leave message.                        |
+| TextOnly\* | Indicates the client is a basic client, made to chat instead of sending locations. Special join/leave message.                    |
+
+\*When specified and used with empty or null `game` in [Connect](#connect), game and game's version validation will be skipped.
```

Comment:
@black-sliver @Exempt-Medic What about something like this instead (splitting these into a second table, with the shared special behaviors listed above?)
(I considered footnotes; putting the details at the bottom of the *entire document* instead of at the bottom of that heading seemed odd to me. I could just move the \* as well, but, this feels a bit clearer/more explicit?)
```suggestion
Tags are represented as a list of strings. Some common client tags follow:

| Name       | Notes                                                                                                                             |
|------------|-----------------------------------------------------------------------------------------------------------------------------------|
| AP         | Signifies that this client is a reference client, its usefulness is mostly in debugging to compare client behaviours more easily. |
| DeathLink  | Client participates in the DeathLink mechanic, therefore will send and receive DeathLink bounce packets.                          |

Additionally, 'Client Type' tags indicate something special about your client. These tags have additional special behaviors:
- When specified and used with empty or null `game` in [Connect](#connect), game and game's version validation will be skipped.
- Displays a special join/leave message for the client. (If multiple Client Type tags are specified, the tag higher on the list below takes priority)

Available Client Type tags:
| Name      | Description                                                                        |
|-----------|------------------------------------------------------------------------------------|
| HintGame  | Indicates the client is a hint game, made to send hints instead of locations.      |
| Tracker   | Indicates the client is a tracker, made to track instead of sending locations.     |
| TextOnly  | Indicates the client is a basic client, made to chat instead of sending locations. |
```

Reply:
What I've been thinking is either

|Name|Description|
|----|-----------|
|HintGame [^1][^2]|Indicates ...|
|Tracker [^1][^2]|Indicates ...|
|TextOnly [^1][^2]|Indicates ...|

[^1]: Adds a special join/leave message
[^2]: For Connect, `game` may be empty or `null`, in which case `game` is not checked

or

|Name|Description|
|----|-----------|
|HintGame|Indicates ... Special join/leave message[^3]. Makes `game` optional[^4]|
|Tracker|Indicates ... Special join/leave message[^3]. Makes `game` optional[^4]||
|TextOnly|Indicates ... Special join/leave message[^3]. Makes `game` optional[^4]||

[^3]: Shows a different message when a player joines or leaves
[^4]: For Connect, `game` may be empty or `null`, in which case `game` is not checked

Not sure if foot notes are actually ideal for the user, alternatively you could go for inline with

|Name|Description|
|----|-----------|
|HintGame|Indicates ... Special join/leave message¹. Makes `game` optional²|
|Tracker|Indicates ... Special join/leave message¹. Makes `game` optional²||
|TextOnly|Indicates ... Special join/leave message¹. Makes `game` optional²||

¹: ...
²: ...

Reply:
The problem with your approach is that there may be a client type tag that does not ignore game or not display a special message

Reply:
The issue with footnotes is that those references at the bottom, will be at the bottom of the ENTIRE document- the whole `.md`- not at the bottom of that heading. So those references are suddenly super far away from the relevant section. If I saw that in docs, I would frankly want to make a PR changing it *away* from what you are suggesting as I would find it confusing.

The manual superscript could work, though, I suppose. Not available rn to make changes though; any chance you could do that as a "suggestion" so I can just click to accept it?

Reply:
I wasn't able to post a suggestion when you asked and then i forgot :sweat_smile: 
Below are 2 possible variations, I don't really have a preference.